frontend/src/api/k8s/dashboardConfig.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  k8sGetResource,
+  k8sPatchResource,
+  k8sUpdateResource,
+} from '@openshift/dynamic-plugin-sdk-utils';
+import { DashboardConfigKind } from '~/k8sTypes';
+import { DASHBOARD_CONFIG } from '~/utilities/const';
+import { ODHDashboardConfigModel } from '~/api/models';
+
+export const getDashboardConfig = (ns: string): Promise<DashboardConfigKind> =>
```

Comment:
Mmmm, I think we talked about this same thing a few weeks ago, this is targeted for the template order, and I think we ended up agreeing on having the order using the pass-through api rather than the backend, and I even tried the backend refresh and was extra slow.

Reply:
Meaning that using this for the template order was more efficient than just talking to the backend

Reply:
Fair, but now our DashboardConfig is in two states in our frontend... this should be corrected next sprint. We can look at what our options are.

Reply:
Follow up issue -- we can talk about it in triage on Monday.
https://github.com/opendatahub-io/odh-dashboard/issues/1201